### ISSUE TITLE
Links & list formatting question

### ISSUE THREAD

**Issue by scottaohara** [mgifford/a11y-courses#12]
Just a thought, but how does anyone feel about reformatting the listings to the following:

- [Title of link](#!): description of link (if applicable)  
- [Title of second link](#!!): description of second link  

or

- [Title of link](#!)  
  description of link (if applicable)  
- [Title of second link](#!!)  
  description of second link  

I'm personally finding it a bit hard to parse, but if no one else is having an issue, then no worries.

**Comment by rbenson** [comment]
I like the second one personally.  The second one will be translated  to like 

      <li><a>link</a><br />description</li>
